package/com.unity.formats.usd/Samples/ExportMeshWithAnimation/ExportMeshWithAnimationExample.cs

```diff
@@ -260,9 +266,11 @@ void LateUpdate()
             m_context.exportMaterials = false;
 
             // Exit once we've recorded all frames.
-            if (m_curFrame > m_frameCount)
+            if (m_curFrame > m_maxFrameCount)
             {
                 StopRecording();
+                IsFinishedRecording = true;
+                AssetDatabase.Refresh();
```

Comment:
I wanted to refresh the Asset Screen right away as soon as the recording part was done in case the user is expecting it as soon as its done recording - but we could move it to do this when the scene playmode is stopped, what do you think?